recipes/prettytable/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: prettytable
+  version: "7"
```

Comment:
Travis-CI has been going for 1:45 min.  The first job finished in just under 10 min.  I really don't know what travis-ci is doing now.  It said that job 2.2 hadn't even started.  So I am at a loss to know what's happening.